scripts/deploy-erc1155-non-transferable.ts

```diff
@@ -0,0 +1,117 @@
+//AssetRegistry not used in demo-app
+//const AssetRegistry = artifacts.require("AssetRegistry");
```

Comment:
Can we please remove these lines from these files?
1. scripts/deploy-erc1155-non-transferable.ts
2. scripts/deploy-protocol-with-ERC1155.ts
3. scripts/deploy-gate.ts

Reply:
Done.
I also renamed of the the files but forgot to check that in before, so pull again if you want to test scripts locally.

Reply:
Can we please remove from `scripts/deploy-gate.ts` as well?

Reply:
sorry, missed that one. I checked it in now.